src/main/js/components/dropdowns/jumplists.js

```diff
@@ -147,6 +110,84 @@ function generateDropdowns() {
   );
 }
 
+function createDropdownContent(element, hasModelLink, hasChildrenLink, href) {
+  return (instance) => {
+    if (element.items) {
+      instance.setContent(Utils.generateDropdownItems(element.items));
+      return;
+    }
+    const sections = {
+      model: null,
+      children: null,
+    };
+
+    const fetchSection = function (urlSuffix) {
+      const response = fetch(Path.combinePath(href, urlSuffix));
+      const json = response.json();
```

Comment:
this is a promise you can't just access it 2 lines down you need to add a `.then()`